src/data-grid/README.md

```diff
@@ -1,34 +1,50 @@
 # Visual Studio Code Data Grid
 
-The `vscode-data-grid` enables developers to display data in a tabular layout. The data grid is created using three components that work together:
+The `vscode-data-grid` enables developers to display data in a tabular layout.
+
+![Data grid hero](/docs/assets/images/data-grid-hero.png)
+
+The data grid is created using three components that work together:
 
 -   `<vscode-data-grid>`: The top level container element.
 -   `<vscode-data-grid-row>`: Displays a single row of data associated with a single record or a header row.
 -   `<vscode-data-grid-cell>`: Displays a single cell of data within a row.
 
-## Data Grid Attributes
+## Usage
+
+| ❌ Don't                                                                     | ✅ Do                                                                      |
+| ---------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| ![Data grid used as a simple list](/docs/assets/images/data-grid-dont-1.png) | ![Data grid with a large data set](/docs/assets/images/data-grid-do-1.png) |
+| Don't use a data grid in place of a basic list.                              | Use data grids to list complex data with secondary information.            |
+
+| ❌ Don't                                                                    | ✅ Do                                                                                           |
+| --------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| ![Data grid in a VS Code sidebar](/docs/assets/images/data-grid-dont-2.png) | ![Data grid in a VS Code editor panel](/docs/assets/images/data-grid-do-2.png)                  |
+| Avoid data grids in small containers.                                       | Place data grids in a main content area to display as much information as possible in one view. |
```

Comment:
`Avoid data grids in small containers` this line is 🔥  as it's a pretty strong stance but am in support of it. We do this in one instance in the Hex editor, not sure if that changes this stance:

<img width="508" alt="CleanShot 2022-02-04 at 11 01 09@2x" src="https://user-images.githubusercontent.com/35271042/152587583-801da3d7-2b07-4215-9f16-16dd0d7d2510.png">